src/Discord.Net.Rest/Entities/Channels/ChannelHelper.cs

```diff
@@ -39,9 +39,17 @@ internal static class ChannelHelper
                     }).ToArray()
                     : Optional.Create<API.Overwrite[]>(),
                 Flags = args.Flags.GetValueOrDefault(),
+                IconEmoji = args.IconEmoji.IsSpecified
+                    ? args.IconEmoji.Value is Emote emote
+                        ? new API.Emoji { Id = emote.Id, Name = emote.Name }
+                        : args.IconEmoji.GetValueOrDefault(null) is Emoji emoji
```

Comment:
Looks like default value never be used here because `IconEmoji.IsSpecified` already checked and passing through `true` branch.

Reply:
Hmm, yeah, nice catch. Idk why I ended up with `GetValueOrDefault`, seems to work fine without it.